tests/test_new_meshing_workflow.py

```diff
@@ -1507,3 +1505,47 @@ def test_switching_workflow_interface(new_mesh_session):
     lw = new_mesh_session.load_workflow(file_path=saved_workflow_path)
     wt2 = new_mesh_session.watertight()
     del wt1, ft, tw, cw, lw, wt2
+
+
+@pytest.mark.codegen_required
+@pytest.mark.fluent_version(">=24.1")
+def test_duplicate_children_of_compound_task(new_mesh_session, mixing_elbow_geometry):
+    watertight = new_mesh_session.watertight()
+    watertight.import_geometry.file_name = mixing_elbow_geometry
+    watertight.import_geometry()
+    watertight.add_local_sizing.add_child_and_update(
+        state={
+            "boi_control_name": "wall",
+            "boi_face_label_list": ["wall-elbow", "wall-inlet"],
+            "boi_size": 10,
+        }
+    )
+    watertight.add_local_sizing.add_child_and_update(
+        state={
+            "boi_control_name": "inlet",
+            "boi_face_label_list": ["wall-inlet", "cold-inlet", "hot-inlet"],
+            "boi_size": 10,
+        }
+    )
+    watertight.add_local_sizing.add_child_and_update(
+        state={
+            "boi_control_name": "outlet",
+            "boi_face_label_list": ["outlet"],
+            "boi_size": 10,
+        }
+    )
+
+    assert {
+        "child_1_of_add_local_sizing",
```

Comment:
Is there a reason for preferring child_n_of_ prefix over _child_n suffix?

Reply:
No, is the latter better? We can update it.

Reply:
Does the _of_ part look a bit awkward in an object name? It's also a bit longer.

Reply:
I agree. Will update the same.